scripts/lib/CIME/case.py

```diff
@@ -1225,6 +1230,33 @@ def create_clone(self, newcase, keepexe=False, mach_dir=None, project=None, cime
         # lock env_case.xml in new case
         lock_file("env_case.xml", newcaseroot)
 
+        # apply user_mods if appropriate
+        newcase_root = newcase.get_value("CASEROOT")
+        if user_mods_dir is not None:
+            if keepexe:
+                # If keepexe CANNOT change any env_build.xml variables - so make a temporary copy of
+                # env_build.xml and verify that it has not been modified
+                shutil.copy(os.path.join(newcaseroot, "env_build.xml"),
```

Comment:
Is it possible that LockedFiles/env_build.xml already exists?   What is the result if that is that case?

Reply:
I have verified that this is not a problem.